amulet/level/formats/anvil_forge_world.py

```diff
@@ -33,5 +39,15 @@ def game_version_string(self) -> str:
         except Exception:
             return f"Java Forge Unknown Version"
 
+    def _register_modded_dimensions(self):
+        for path, directories, files in os.walk(os.path.join(self.path, "dimensions")):
+            if set(("data", "entities", "poi", "region")).issubset(set(directories)):
+                dimension_path_split = path.split(os.sep)
+                dimensions_directory_index = dimension_path_split.index("dimensions")
```

Comment:
You compute the `relpath` further down. The indexes will be constant in that output.